skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -24,3 +24,4 @@ skaffold.configuration.label=Skaffold configuration:
 
 skaffold.no.file.selected.error=Skaffold configuration file is not selected.
 skaffold.invalid.file.error=Skaffold configuration file doesn't exist or invalid.
+skaffold.corrupted.run.settings=Your Skaffold run configuration is corrupted. Please edit or re-create it to fix.
```

Comment:
I wonder if editing it would do much to help? If not, then maybe just suggest to re-create it.